components/docs/docs.css

```diff
@@ -0,0 +1,102 @@
+/* navbar  */
+.docs {
+  background-color: var(--secondary);
+}
+/* sidebar  */
+.components-container {
+  display: grid;
+  /* grid-auto-flow: column; */
+  grid-template-columns: 1fr 5fr;
+}
+.component-connect-container {
+  justify-content: space-between;
+  height: calc(100vh - 7rem);
+  transition: all 0.2s ease;
+  background-color: var(--secondary);
+}
+
+.components-menu {
+  padding: 2rem;
+  height: 80%;
+  overflow-y: scroll;
+}
+.components-menu::-webkit-scrollbar {
+  display: none;
+}
+.component-heading {
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  color: white;
+}
+.component-heading::after {
+  content: "";
+  border-right: 4px solid var(--primary);
+  margin: 1rem;
+}
+.component-heading::before {
+  content: "";
+  border-right: 4px solid var(--primary);
+  margin: 1rem;
+}
```

Comment:
If the content is the same for both of them then use a comma separator.  
`.component-heading::before, .component-heading::after {  .......  }`

Reply:
Thanks